test/test_prototype_transforms_functional.py

```diff
@@ -1337,3 +1346,42 @@ def _compute_expected_bbox(bbox, output_size_):
         else:
             expected_bboxes = expected_bboxes[0]
         torch.testing.assert_close(output_boxes, expected_bboxes)
+
+
+def test_correctness_center_crop_segmentation_mask_on_fixed_input(device):
+    mask = torch.ones((1, 6, 6), dtype=torch.long, device=device)
+    mask[:, 1:5, 2:4] = 0
+
+    out_mask = F.center_crop_segmentation_mask(mask, [2])
+    expected_mask = torch.zeros((1, 4, 2), dtype=torch.long, device=device)
+    torch.testing.assert_close(out_mask, expected_mask)
+
+
+@pytest.mark.parametrize("output_size", [[4, 3], [4]])
+def test_correctness_center_crop_segmentation_mask(output_size):
```

Comment:
I think we could merge these two tests into one with parameters `device` and `output_size`.

Reply:
Ok. I thought that having two test (one with fixed input + device and one with random input) was a sort of scheme to follow. That's why I kept them separated.